Sources/docc/DocCDocumentation.docc/snippets.md

```diff
@@ -0,0 +1,144 @@
+# Adding Code Snippets to your Content
+
+Create and include code snippets to illustrate and provide examples of how to use your API.
+
+## Overview
+
+...tbd...
+- Describe the problem and summarize the developer action.
+- Explain why the problem is relevant and provide context for the task. Don't simply repeat the abstract; expand on it.
+- Keep the Overview to one or two paragraphs.
+- For very short articles that consist of just a couple of paragraphs, all of the content can be in the Overview.
+
+...tbd...
```

Comment:
It might be good to assume that the reader doesn't know the destination between a code snippet and a markdown code block element and lead with that information. A reader who finds this information might be overwhelmed of all the steps if all they want to do is to write a code block in their markup. That distinction might serve as a good segue to the benefits of compiled code examples which lets the reader make an informed decision about using code blocks or code snippets in their documentation.